workspaces/global-header/packages/app/src/App.tsx

```diff
@@ -137,6 +139,15 @@ const routes = (
 
 export default app.createRoot(
   <>
+    <GlobalStyles
+      styles={{
+        html: { overflowY: 'hidden' },
+        ':root': {
+          '--rhdh-v1-page-inset': '1.5rem',
+          '--rhdh-v1-header-height': '64px',
```

Comment:
maybe so it's more clear that this env is set in only?

```suggestion
          '--global-header-dev-header-height': '64px',
```

Reply:
@ciiay, let me if all of @logonoff comments you have addressed, so that I can review it.

Reply:
I have replaced `--rhdh-v1-header-height` with `--global-header-default-height`.